sympy/geometry/line.py

```diff
@@ -385,26 +435,23 @@ def direction(self):
         return self.p2 - self.p1
 
     def intersection(self, other):
-        """The intersection with another geometrical entity.
+        """Generate the intersection with another geometrical entity.
```

Comment:
I don't think Generate is the right word here. Also, please move this to the next line.